components/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Image from "next/image";
+
+const Card = ({ item }) => {
+  return (
+    <div className="flex justify-center mt-5">
```

Comment:
use width to make the div responsive. See https://tailwindcss.com/docs/max-width